explorer/src/utils/chainStateParsing.ts

```diff
@@ -0,0 +1,53 @@
+import type { StorageKey } from '@polkadot/types'
+import type { AnyTuple, Codec } from '@polkadot/types-codec/types'
+import { Deposit, Operators, RawDeposit, Withdrawal } from 'types/consensus'
+
+export const formatOperators = (
+  operators: [StorageKey<AnyTuple>, Codec][],
+  operatorIdOwner: [StorageKey<AnyTuple>, Codec][],
+) =>
+  operators.map((operator, key) => {
+    const op = operator[1].toJSON() as Omit<Operators, 'id' | 'operatorOwner'>
+    return {
+      id: (operator[0].toHuman() as string[])[0],
+      operatorOwner: operatorIdOwner[key][1].toJSON() as string,
+      ...op,
+      minimumNominatorStake: BigInt(op.minimumNominatorStake).toString(10),
+      currentTotalStake: BigInt(op.currentTotalStake).toString(10),
+      currentTotalShares: BigInt(op.currentTotalShares).toString(10),
+      totalStorageFeeDeposit: BigInt(op.totalStorageFeeDeposit).toString(10),
+      status: JSON.stringify(op.status),
+    } as Operators
+  })
+
+export const formatDeposits = (deposits: [StorageKey<AnyTuple>, Codec][]) =>
+  deposits.map((deposit) => {
+    const parsedDeposit = deposit[1].toJSON() as RawDeposit
+    return {
+      operatorId: parseInt((deposit[0].toHuman() as string[])[0]),
+      account: (deposit[0].toHuman() as string[])[1],
+      shares: parseInt(parsedDeposit.known.shares.toString(), 16).toString(),
```

Comment:
This may be the source of scientific notation on `deposit.shares` mentioned above. Is there a reason to use `parseInt` here instead of `BigInt` like other big integers in this function? Using the suggested code works in local testing.
```suggestion
      shares: BigInt(parsedDeposit.known.shares).toString(10),
```